src/python_testing/TC_CLCTRL_3_1.py

```diff
@@ -254,7 +251,6 @@ async def test_TC_CLCTRL_3_1(self):
                 logging.info(f"OverallCurrentState: {overall_current_state}")
 
                 CurrentLatch = overall_current_state.latch
-                asserts.assert_in(CurrentLatch, [True, False], "OverallCurrentState.latch is not in the expected range")
             else:
                 asserts.assert_true(False, "OverallCurrentState attribute is not supported.")
```

Comment:
we can use `asserts.fails(...)` to simplify these calls - applies in multiple location.